### DISCUSSION TITLE
TextFsmParse accept both fs path and URL

### DISCUSSION THREAD

**Discussion by networkop** [scrapli/scrapligo#62] [Ideas]
I thought it'd be cool to have `TextFsmParse` recognise URL schemas in input string as well as the local filesystem path.
This would make it easier to plug in textfsm templates from https://github.com/networktocode/ntc-templates for example.
 wdyt?

**Comment by carlmontanari** [comment]
I think it sounds like a great idea! Interested in putting a PR together? I wonder if/how many ext deps this would require? Ideally none is the amount I'd love haha, but defo interested in this regardless!

**Comment by networkop** [comment]
yep, I can do a PR. I'd assume we'd need to pull in some http-related deps to download the yaml and that's it.

**Reply by carlmontanari** [reply]
Cool, sounds great! wonder if net/http client would work? I just realized I've used that for health check stuff in boxen and for at least that ultra simple use case it works well enough, might be enough for this too!
